### PR TITLE
cleanup(misc): migrate from chalk to picocolors

### PR DESCRIPTION
**Pull request by 43081j**
Migrates from `chalk` to the much smaller and faster `picocolors`.

Part of a wider ecosystem cleanup, we are moving many projects over to picocolors or node's native styleText. 

If you let me know you're happy with this change, I'll push a branch for each of the other packages to do the same. 

I will catch up from master when I'm back on my laptop 👍

### CODE REVIEW

File: packages/create-nx-plugin/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "homepage": "https://nx.dev",
   "dependencies": {
     "create-nx-workspace": "file:../create-nx-workspace",
-    "chalk": "^4.1.0",
+    "picocolors": "^1.1.0",
```
Comment (JamesHenry) [inline packages/create-nx-plugin/package.json:33]: The repo doesn't use workspaces, so please add this to the root package.json too

**Comment by JamesHenry** [comment]
Thanks @43081j this is very similar in nature to the recently merged: https://github.com/nrwl/nx/pull/27932

Please can you implement the same feedback I gave on there - add the ESLint rule to this project to catch imports of chalk and give the feedback that picocolors should be used instead

File: packages/create-nx-plugin/package.json (hunk 1 above)
Comment (JamesHenry) [reply packages/create-nx-plugin/package.json:33]: This is at least in part why your CI run failed

**Review by JamesHenry — CHANGES_REQUESTED** [review]
See comments

**Comment by JamesHenry** [comment]
This change should really be `cleanup`, it should be imperceptible to the end user and not fix a bug nor add a feature.

File: packages/create-nx-plugin/package.json (hunk 1 above)
Comment (43081j) [reply packages/create-nx-plugin/package.json:33]: Ah I somehow totally missed that. Had been wondering what kind of setup it was meant to be

Thanks for the explanation

**Comment by 43081j** [comment]
Makes sense to me 👍 

Wasn't sure what the correct scope and type of change was. Thanks for the tip

Will resolve the comments soon

**Comment by 43081j** [comment]
I'm struggling to understand the current failure

any chance you could help me out? 👀  I think its a failing test but I cant seem to find where in the circleci output we can get hold of which test

**Comment by JamesHenry** [comment]
@43081j I believe it was a memory issue, I have fixed up the lock file conflict and pushed

**Review by JamesHenry — APPROVED** [review]

**Comment by JamesHenry** [comment]
Thanks again!

**Pull request merged**